storage/src/adb/current.rs

```diff
@@ -1281,200 +1227,10 @@ pub mod test {
-
-                // Commit periodically to advance the inactivity floor
-                if i % 100 == 99 {
-                    db.commit().await.unwrap();
-                }
-            }
-
-            // Final commit to establish the inactivity floor
-            db.commit().await.unwrap();
-
-            // Get the root digest
-            let original_root = db.root(&mut hasher).await.unwrap();
-
-            // Verify the pruning boundary is correct
-            let oldest_retained = db.oldest_retained_loc().unwrap();
-            let inactivity_floor = db.any.inactivity_floor_loc;
-            assert_eq!(
-                oldest_retained,
-                inactivity_floor.saturating_sub(db_config.pruning_delay)
-            );
-
-            // Get proof of items below inactivity floor but after pruning boundary
-            let proof_start = oldest_retained;
-            let proof_end = std::cmp::min(inactivity_floor, oldest_retained + 10);
-            let max_ops = proof_end - proof_start;
-
-            let (original_proof, original_ops, original_chunks) = db
-                .range_proof(hasher.inner(), proof_start, max_ops)
-                .await
-                .unwrap();
-
-            // Verify the proof works
-            assert!(CurrentTest::verify_range_proof(
-                &mut hasher,
-                &original_proof,
-                proof_start,
-                &original_ops,
-                &original_chunks,
-                &original_root
-            ));
-
-            // Close and reopen the database
-            db.close().await.unwrap();
-            let db = CurrentTest::init(context.clone(), db_config).await.unwrap();
-
-            // Confirm root is identical after restart
-            let reopened_root = db.root(&mut hasher).await.unwrap();
-            assert_eq!(original_root, reopened_root);
-
-            // Get proof of items below inactivity floor again
-            let (reopened_proof, reopened_ops, reopened_chunks) = db
-                .range_proof(hasher.inner(), proof_start, max_ops)
-                .await
-                .unwrap();
-
-            // Verify the proof still works and is identical
-            assert_eq!(original_proof.size, reopened_proof.size);
-            assert_eq!(original_proof.digests, reopened_proof.digests);
-            assert_eq!(original_ops, reopened_ops);
-            assert_eq!(original_chunks, reopened_chunks);
-
-            assert!(CurrentTest::verify_range_proof(
-                &mut hasher,
-                &reopened_proof,
-                proof_start,
-                &reopened_ops,
-                &reopened_chunks,
-                &reopened_root
-            ));
-
-            db.destroy().await.unwrap();
-        });
-    }
-
-    /// Test that databases with different `pruning_delay` values generate the same root.
-    #[test_traced("WARN")]
-    pub fn test_current_db_different_pruning_delays_same_root() {
```

Comment:
It would be great to retain this test where we call prune on two different dbs with different values and ensure the root is identical.

Reply:
done